Doc/library/socket.rst

```diff
@@ -1079,6 +1079,15 @@ to sockets.
    to decode C structures encoded as byte strings).
 
 
+.. method:: socket.getblocking()
+
+   Return ``True`` if socket is in blocking mode, ``False`` if in
+   non-blocking.  This reflects the last call to :meth:`setblocking`
+   or :meth:`settimeout`.
```

Comment:
"This reflects the last call to :meth:`setblocking`  or :meth:`settimeout`."

What abou the following case?

```
>>> import socket; socket.socket(type=socket.SOCK_STREAM | socket.SOCK_NONBLOCK).getblocking()
False
```

I suggest to remove the second sentence.

Reply:
Sure